packages/core/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,24 @@
+import { ExpandMoreIcon } from '@medly-components/icons';
+import React, { SFC, useState } from 'react';
+import Text from '../Text';
+import { Content, Wrapper, Section } from './Accordion.styled';
+import { Props } from './types';
+
+
+export const Accordion: SFC<Props> = React.memo(({ title, titleColor, bgColor, children }) => {
+    const [active, setActive] = useState(false),
+        toggleAccordion = () => setActive(active ? false : true);
```

Comment:
Use it like this -> toggleAccordion = useCallback(() => setActive(val => !val), []);

Reply:
done